app/src/main/java/org/wikipedia/travel/MainPlannerFragment.java

```diff
@@ -154,10 +158,14 @@ public void onRequestTripListUpdate() {
         updateUserTripList();
     }
 
+    @Override
+    public void onRequestDestinationListUpdate() {
+        updateUserDestinationList();
+    }
 
     @Override
     public void onDeleteTrip(long id) {
-        CallbackTask.execute(() -> TripDbHelper.instance().deleteList(getTrip(id)), new CallbackTask.DefaultCallback<Object>() {
+        CallbackTask.execute(() -> TripDbHelper.instance().deleteList(getTrip(id)), new CallbackTask.DefaultCallback <Object>() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['<' is preceded with whitespace.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1526985)